pycbc/io/inference_hdf.py

```diff
@@ -496,26 +497,33 @@ def write_data(self, strain_dict=None, stilde_dict=None,
         if strain_dict is not None:
             self.write_strain(strain_dict, group=group)
 
-    def write_injections(self, injection_file, ifo):
+    def write_injections(self, injection_files, ifo):
         """ Writes injection parameters for an IFO to file.
 
         Parameters
         ----------
-        injection_file : str
+        injection_files : str
             Path to HDF injection file.
         ifo : str
             IFO name.
         """
-        subgroup = "{ifo}/injections"
-        self.create_group(subgroup.format(ifo=ifo))
-        try:
-            with h5py.File(injection_file, "r") as fp:
-                for param in fp.keys():
-                    self[subgroup.format(ifo=ifo)][param] = fp[param][:]
-                for key in fp.attrs.keys():
-                    self[subgroup.format(ifo=ifo)].attrs[key] = fp.attrs[key]
-        except IOError:
-            logging.warn("Could not read %s as an HDF file", injection_file)
+        if not isinstance(injection_files, list):
+            injection_files = [injection_files]
+        for i, injection_file in enumerate(injection_files):
+            try:
+                with h5py.File(injection_file, "r") as fp:
+                    injection_file_name = os.path.basename(injection_file)
+                    injection_file_tag = "inj_file" + str(i)
```

Comment:
``inj_file0`` looks weird compared to ``inj_file_0`` or ``0``.

Reply:
The injection jobs in our workflows have labels like inj_job0 etc, so I named this something like that. But yeah.. it does not look good. I'll change it to `inj_file_0` then, if that looks okay. I just thought of using `inj_file` instead of simply the index so that it is clear that it's named based on the injection file number, and not be confused with like job number or something.